core/src/player.rs

```diff
@@ -148,7 +160,7 @@ impl<'gc> GcRootData<'gc> {
     }
 }
 
-make_arena!(GcArena, GcRoot);
+make_arena!(pub GcArena, GcRoot);
```

Comment:
I decided to just do StaticCallstack, I didn't want to name it specific to AVM2 in case we ever want to add something similar to AVM1. Its called "Static" to indicate it's not bounded by any lifetime and can be moved freely, unlike the regular callstack.